docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -38,16 +39,17 @@ aws iam create-login-profile --user-name ${ADMIN} --password ${INITIAL_PASSWORD}
 echo Now log in at ${CONSOLE_URL}  with user name ${ADMIN} password ${INITIAL_PASSWORD} and change the password.
 ----
 
-== Filter Actor
```

Comment:
I got rid of the term 'Actor'.   I'm now referring to the entity within the KMS (Token in the case of Vault, IAM User in the case of AWS).